### PR TITLE
Minor enhancements to contact fields, name field

### LINKED ISSUE #208: Add a "title" parameter for name_fields(); suffix should be optional
In some cultures "title" is commonly used and "suffix" is not. It would be nice to add this flexibility to the ALIndividual.name_fields() method.

### LINKED ISSUE #216: Make the contact_methods method translatable -- probably mirror the Mako template method
https://github.com/SuffolkLITLab/docassemble-AssemblyLine/blob/9ff928d502010b736feb23a63b87092c4b709929/docassemble/AssemblyLine/al_general.py#L142

### PR DESCRIPTION
**Pull request by nonprofittechy**
1. Translatable contact method fields, fix #216 
2. Add a new, optional "title" parameter when using name fields. Fix #208

### CODE REVIEW

File: docassemble/AssemblyLine/data/questions/ql_baseline.yml [hunk 1]
```diff
@@ -2309,4 +2314,21 @@ content: |
...
-  Self described pronouns
\ No newline at end of file
+  Self described pronouns
+---
+############# Contact method strings ################
+---
+generic object: ALIndividual
+template: x.phone_number_contact_label
+content: |
+  by phone at
```
Comment (BryceStevenWilley) [inline docassemble/AssemblyLine/data/questions/ql_baseline.yml:2324]: I think we want:

```suggestion
content: >-
  by phone at
```

here, right? Haven't tested, but I always have to check https://yaml-multiline.info/ anway

File: docassemble/AssemblyLine/data/questions/ql_baseline.yml [hunk 2]
```diff
@@ -2309,4 +2314,21 @@ content: |
...
 content: |
-  Self described pronouns
\ No newline at end of file
+  Self described pronouns
+---
+############# Contact method strings ################
+---
+generic object: ALIndividual
+template: x.phone_number_contact_label
+content: |
```
Comment (nonprofittechy) [inline docassemble/AssemblyLine/data/questions/ql_baseline.yml]:
```suggestion
content: >-
```

File: docassemble/AssemblyLine/data/questions/ql_baseline.yml [hunk 3]
```diff
@@ -2309,4 +2314,21 @@ content: |
...
+  by phone at
+---
+generic object: ALIndividual
+template: x.email_contact_label
+content: |
```
Comment (nonprofittechy) [inline docassemble/AssemblyLine/data/questions/ql_baseline.yml:2328]:
```suggestion
content: >-
```

File: docassemble/AssemblyLine/data/questions/ql_baseline.yml [hunk 4]
```diff
@@ -2309,4 +2314,21 @@ content: |
...
+  by email at
+---
+generic object: ALIndividual
+template: x.other_contact_method_label
+content: |
```
Comment (nonprofittechy) [inline docassemble/AssemblyLine/data/questions/ql_baseline.yml:2333]:
```suggestion
content: >-
```

File: docassemble/AssemblyLine/data/questions/ql_baseline.yml (hunk 1 above)
Comment (nonprofittechy) [reply docassemble/AssemblyLine/data/questions/ql_baseline.yml]: Great catch, thanks!

**Pull request merged**